src/requestLogging/README.md

```diff
@@ -5,13 +5,18 @@
 This add-on facilitates logging information about requests and responses.
 It's intended to work with an app-provided logger such as [pino](http://getpino.io/) or [Bunyan](https://github.com/trentm/node-bunyan).
 
-It provides two main features:
-
-- [`contextFields`](#context-fields) returns log fields related to the incoming request.
+It provides a few main features:
 
+- [`createLoggerContext, getLoggerContext, createLoggerContextMiddleware, contextFields`](#context-logging) auto inject request based information into a logger's context
 - [`createMiddleware`](#request-log) creates a Koa middleware for logging request and response information
 
-## Context Fields
+## Context Logging
```

Comment:
Do you think it would be useful to expose a more minimal interface for context logging, at least in the typical case? This interface as-is grants the user flexibility to work with the underlying `AsyncLocalStorage` but it doesn't seem to present much value on top of it.

What if we worked backwards from and poked holes in this hypothetical:

```typescript
const middleware = RequestLogging.createMiddleware(() => undefined);

const { mixin } = middleware;

const logger = pino({
  mixin,
});

new Koa().use(middleware).use((ctx) => {
  logger.info('I have context!');
});
```

Reply:
Hmmm I like it. I started with initialising one in the global state and I realised oh that's not great. I'll give this a shot